code/game/turfs/simulated/floor.dm

```diff
@@ -57,26 +56,26 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 		return
 	switch(severity)
 		if(1.0)
-			src.ChangeTurf(/turf/space)
+			ChangeTurf(baseturf)

```

Comment:
So these baseturf changes are so that Lavaland doesn't get exposed to space. We need to make sure this doesn't cause any issues elsewhere, but I don't think it will.

We do still use /tg/'s old system of mapping_helper baseturf setters, instead of a ZTRAIT based system. But to use that, I'd also have to port over the ZTRAITs, which would break the scope of this PR even further.